MotorControl/MotorControl/Resources/BalanceSectionStep.json

```diff
@@ -0,0 +1,105 @@
+{
+    "identifier": "balance",
+    "type": "section",
+    "asyncActions": [{
+        "identifier": "motion",
+        "type": "motion",
+        "requiresBackgroundAudio": true,
+        "recorderTypes": ["accelerometer", "gyro", "gravity", "userAcceleration", "attitude", "rotationRate"],
+        "startStepIdentifier": "motion",
+        "stopStepIdentifier": "motion"
+    }],
+    "progressMarkers":["phoneInPocket","motion"],
+    "steps": [{
+        "identifier": "turnAroundInstructions",
+        "type": "instruction",
+        "title": "Turn around, and then stand still",
+        "text": "You’ll turn around a circle and stand still, with your feet shoulder-width apart.\n\nRest your arms by your side and try to avoid moving for 30 seconds.",
```

Comment:
```suggestion
        "text": "You’ll turn around in a circle and stand still, with your feet shoulder-width apart.\n\nRest your arms by your side and try to avoid moving for 30 seconds.",
```